src/Redux/configureStore.js

```diff
@@ -1,10 +1,11 @@
 import { configureStore } from '@reduxjs/toolkit';
+// import thunk from 'redux-thunk';
```

Comment:
- [ ] please try to remove the unused commented code to keep the file clean and pro